runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -4756,9 +4756,15 @@ MM_CopyForwardScheme::verifyClassObjectSlots(MM_EnvironmentVLHGC *env, J9Object
 	J9Class *classPtr = J9VM_J9CLASS_FROM_HEAPCLASS((J9VMThread*)env->getLanguageVMThread(), classObject);
 
 	if (NULL != classPtr) {
-		volatile j9object_t * slotPtr = NULL;
-
+		volatile j9object_t * slotPtr = NULL;	
 		do {
+			/*
+			 * scan default values
+			 */
+			if(J9_IS_J9CLASS_VALUETYPE(classPtr) && (!J9CLASS_IS_ARRAY(classPtr))){
+				verifyMixedObjectSlots(env, (J9Object *) &(classPtr->flattenedClassCache->clazz));
+			}
+			
```

Comment:
This is not needed, and in fact, is incorrect.

The new slot you added to the class is of type class slot (as opposed to object slot).  verifyMixedObjectSlots expects (mixed) **object** slot provided.

Reply:
No problem. I'll remove that.